docs/guide/persistence_cache.md

```diff
@@ -202,7 +202,8 @@ With that in mind, the following configurations of Redis are possible:
     - Typically used with a load balancer (e.g. HAproxy) in the front in order to only speak to elected master
         - An alternative is that application logic itself speaks to Sentinel in order to always ask for elected master before talking to cache.
 
-For best performance we recommend use of Redis Sentinel if it fits your needs. However different cloud providers have managed services that are easier to set up, and might perform better. Notable Services:
+For best performance we recommend use of Redis Sentinel if it fits your needs. But know that you can not setup RedisAdapter to make Platform 2.5 communicate with Sentinel since Symfony 3 does not support Redis sentinel. We are not aware of any bundle that can allow you to perform this case.
+Feel free to use a custom solution in order to make it work. However different cloud providers have managed services that are easier to set up, and might perform better. Notable Services:
```

Comment:
Actually I see now that the point above is supposed to cover this, so this is maybe what we need to make more clear:

>     - Typically used with a load balancer (e.g. HAproxy) in the front in order to only speak to elected master
>        - An alternative is that application logic itself speaks to Sentinel in order to always ask for elected master before talking to cache.

So maybe we can simplify to:
```diff
     - Typically used with a load balancer (e.g. HAproxy) in the front in order to only speak to elected master
         - An alternative is that application logic itself speaks to Sentinel in order to always ask for elected master before talking to cache.
+        - In Symfony 5 this is done for you with support for Redis Sentinel in Redis Adapters theme-selves, but for 2.5 you'll need to do this yourself.

- For best performance we recommend use of Redis Sentinel if it fits your needs. However different cloud providers have managed services that are easier to set up, and might perform better. Notable Services:
+ Several cloud providers have managed services that are easier to set up, handles replication and scalability for you, and might perform better. Notable Services:
```

Reply:
@andrerom sounds good. One point, If I'm not mistaken, Redis Sentinel started to be supported from Symfony 4.
https://symfony.com/blog/new-in-symfony-4-4-redis-cache-improvements#added-redis-sentinel-support
We might turn Symfony 5 into [Symfony 4.4 ](https://symfony.com/blog/new-in-symfony-4-4-redis-cache-improvements#added-redis-sentinel-support)with the link on it .

Reply:
Well we don't have any versions supporting symfony 4, so maybe better to say eZ Platform 3 then